apps/sel4test-tests/src/tests/schedcontext.c

```diff
@@ -46,6 +46,14 @@ int test_sched_control_configure(env_t env)
     error = api_sched_ctrl_configure(simple_get_sched_ctrl(&env->simple, 0), sc, 5000llu, 1000llu, 0, 0);
     test_eq(error, seL4_RangeError);
 
+    /* test budget < MIN_BUDGET doesn't work */
+    error = api_sched_ctrl_configure(simple_get_sched_ctrl(&env->simple, 0), sc, 1, 5000, 0, 0);
```

Comment:
Not that it has very different semantics in this case, but the other API calls use `5000llu` instead of `5000` etc. It'd be nice to keep those consistent.

Reply:
That's what I started with, but then I ended up with ``1llu`` and that's just too ugly. I can't think of a good reason why llu would improve anything other than consistency, it just makes things less readable. Okay if I remove the others instead?